euler/005/problem.day

```diff
@@ -46,7 +46,7 @@ fun _recursive_multiply (upper_bound: int arr: array<int> len: size_t index: siz
 end
 
 fun _problem005 () int is
-	let primes: array<int> <- [
+	mut primes: array<int> <- [

```

Comment:
Necessary because function parameters are non-const by default for now. That means that passing arrays to functions requires that the array be mutable for now